src/_content/pages/errorpage.njk

```diff
@@ -0,0 +1,39 @@
+---
+title: 404 not found
+permalink: /error-page/index.html
+---
```

Comment:
Lets make use of the markdown layout, similar to what you see here on the Discord Success page. https://raw.githubusercontent.com/alexcarpenter/frontend-foundations/master/src/_content/pages/discord-success.md

I think we could keep it really simple at the moment. Below is what you could add.

```yaml
---
layout: markdown
title: 404
description: Page not found
---
```